.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
           path: |
             ${{ env.pythonLocation }}
             ~/.cache/pypoetry
-          key: ${{ env.pythonLocation }}-${{ hashFiles('metricflow/poetry.lock') }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('poetry.lock') }}-ubuntu-latest-22.04
```

Comment:
Well, it is not detailed enough.. considering `ubuntu-latest` could change (unlikely it will change soon), should we pin the version to `ubuntu-22.04` as well ?

runner.os | string | The operating system of the runner executing the job. Possible values are Linux, Windows, or macOS.
-- | -- | --

Reply:
My thinking with this setup was we'd use ubuntu-latest and tag the version in the cache key, then update the version in the cache key if an ubuntu upgrade causes these problems in the future.

Reply:
I don't see anything in GitHub's contexts that allow us the granularity we need, so unless we feel strongly about pinning versions I'd like to roll with this. We can always change it later!